lib/mechanize/form.rb

```diff
@@ -365,7 +374,12 @@ def request_data
 
   # Removes all fields with name +field_name+.
   def delete_field!(field_name)
-    @fields.delete_if{ |f| f.name == field_name}
+    @fields.delete_if { |f| f.name == field_name}
```

Comment:
Thank you. Personally I think there should be a space at the end too, but it seems like inconsistency is the norm for now.